README.md

```diff
@@ -6,20 +6,29 @@ A secure, shared workspace for secrets.
 [Documentation](https://torus.sh/docs) |
 [Twitter](https://twitter.com/toruscli) |
 [Security Disclosure](https://github.com/manifoldco/torus-cli/blob/master/docs/security.md) |
-[Code of Conduct](./.github/CONDUCT.md) |
-[Contribution Guidelines](./.github/CONTRIBUTING.md)
+[Code of Conduct](https://github.com/manifoldco/torus-cli/blob/master/.github/CONDUCT.md) |
+[Contribution Guidelines](https://github.com/manifoldco/torus-cli/blob/master/.github/CONTRIBUTING.md)
 
 [![Travis](https://img.shields.io/travis/manifoldco/torus-cli/master.svg)](https://travis-ci.org/manifoldco/torus-cli)
 [![Go Report Card](https://goreportcard.com/badge/github.com/manifoldco/torus-cli)](https://goreportcard.com/report/github.com/manifoldco/torus-cli)
 [![npm](https://img.shields.io/npm/v/torus-cli.svg)](https://www.npmjs.com/package/torus-cli)
-[![License](https://img.shields.io/badge/license-BSD-blue.svg)](./LICENSE.md)
+[![License](https://img.shields.io/badge/license-BSD-blue.svg)](https://github.com/manifoldco/torus-cli/blob/master/LICENSE.md)
 
-![](./graphic.png)
+![](https://github.com/manifoldco/torus-cli/blob/master/graphic.png)
```

Comment:
npm doesn't auto resolve this using the `repository` field in the package.json? 😮